src/NzbDrone.Core/IndexerProxies/FlareSolverr/FlareSolverr.cs

```diff
@@ -149,15 +170,16 @@ private HttpRequest GenerateFlareSolverrRequest(HttpRequest request)
                 throw new FlareSolverrException("Unsupported HttpMethod: " + request.Method);
             }
 
+            InjectProxy(req);
+
             var apiUrl = string.Format("{0}/v1", Settings.Host.TrimEnd('/'));
             var newRequest = new HttpRequest(apiUrl, HttpAccept.Json);
 
             newRequest.Headers.ContentType = "application/json";
             newRequest.Method = HttpMethod.Post;
-            newRequest.LogResponseContent = true;
             newRequest.SetContent(req.ToJson());
 
-            _logger.Debug("Cloudflare Detected, Applying FlareSolverr Proxy {0} to request {1}", Name, request.Url);
+            _logger.Debug("Applying FlareSolverr Proxy {0} to request {1}", Name, request.Url);
```

Comment:
Same here

Reply:
The files got corrupted to binary after my computer restarted

So I copied the old changes